wagtail/wagtailcore/utils.py

```diff
@@ -37,10 +37,12 @@ def resolve_model_string(model_string, default_app=None):
         return model_string
 
     else:
-        raise LookupError("Can not resolve {0!r} into a model".format(model_string), model_string)
+        raise ValueError("Can not resolve {0!r} into a model".format(model_string), model_string)
 
```

Comment:
Perhaps worth adding something like this to the docstring:
"Raises ValueError if the model_string is invalid. Raises a LookupError if the model_string cannot be found in the project"